test_ppod.py

```diff
@@ -99,3 +101,29 @@ def test_filter_files_in_bucket_without_matching_file(mocked_s3):
     with pytest.raises(KeyError):
         files = filter_files_in_bucket("ppod", "download/")
         next(files)
+
+
+def test_post_files_to_pod_success(mocked_pod):
+    with open("fixtures/pod_with_namespaces.xml", "rb") as pod_with_namespaces, open(
+        "fixtures/pod_response.html", "r"
+    ) as pod_response:
+        response = post_files_to_pod(
+            "http://example.example/organizations/ORG/uploads?stream=default",
+            {"Authorization": "Bearer 1234abcd"},
+            "pod_file",
+            pod_with_namespaces,
+        )
+        assert response.status_code == 200
+        assert response.text == pod_response.read()
```

Comment:
Since we don't use the response text at all in the code, I'm not sure it makes sense to check its content in the test?